legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-ansi/src/test/java/org/finos/legend/engine/persistence/components/ingestmode/bitemporal/BitemporalDeltaWithBatchIdTest.java

```diff
@@ -1503,6 +1683,13 @@ void testMilestoningSourceSpeciesFromWithDeleteIndicatorWithDataSplitFilterDupli
         Assertions.assertEquals(getExpectedCleanupSql(tempWithDeleteIndicatorName, "legend_persistence_tempWithDeleteIndicator"), operations.get(0).ingestSql().get(9));
         Assertions.assertEquals(getExpectedCleanupSql(stageWithoutDuplicatesName, "legend_persistence_stageWithoutDuplicates"), operations.get(0).ingestSql().get(10));
 
+        String incomingRecordCount = "SELECT COUNT(*) as incomingRecordCount FROM \"mydb\".\"staging\" as stage WHERE (stage.\"data_split\" >= 2) AND (stage.\"data_split\" <= 5)";
```

Comment:
We can have a placeholder for lower and upper bound and then replace it with values.
That way we can reuse the same set of SQL.